.github/CODEOWNERS

```diff
@@ -3,7 +3,11 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @jsf9k
+* @dav3r @jsf9k @mcdonnnj
+
+# These folks own any files in the .github directory at the root of
+# the repository and any of its subdirectories.
+/.github/ @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
This stuff should go at the very end of this file to ensure that it supersedes anything above it.

Reply:
Whoops- I accidentally duplicated that block.  Cleaned up with https://github.com/cisagov/nessus-packer/pull/20/commits/d20d1ad6749748a3eb80c08d79ce8d49f4b2f149.  Thanks for catching that!

Reply:
Thanks.  I didn't notice that it was duplicated - I only noticed the block in the wrong place.